addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4154,6 +4154,82 @@ registry.ReplaceMedia = SnippetOptionWidget.extend({
     }
 });
 
+/**
+ * Options of a font awesome icon.
+ */
+registry.FontAwesome = SnippetOptionWidget.extend({
```

Comment:
Given that there will be a few common commands between FontAwesome and Image, maybe I should rename this MediaTools and reuse it when I do images.